### PR TITLE
Created "Move From" and "Move To" buttons. Renamed buttons to remove …

### PR DESCRIPTION
**Pull request by terri-hewitt**
…card.

### CODE REVIEW

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 1]
```diff
@@ -89,9 +90,14 @@
 </table>
 
 <button id="dealButton">Deal</button>
-<BR>Remove from: <button id="Remove1">Column1</button><button id="Remove2">Column2</button><button id="Remove3">Column3</button><button id="Remove4">Column4</button>
```
Comment (elitucker) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: If we are going with the buttons at the bottom of the columns idea, we probably want to put them in the same div as the table that holds the cards.

Comment (terri-hewitt) [reply src/main/java/views/AcesUp/AcesUp.flt.html]: Ok! I'll go make that change and update the pull request.

Comment (terri-hewitt) [reply src/main/java/views/AcesUp/AcesUp.flt.html]: All right, I the buttons are now located in two extra rows I added to the card table. The "Move From" and "Move To" buttons share the last cell. The ID for the button cells are: "ButtonLocation 1" (Remove button) and "ButtonLocation 2" (Move from/to buttons).

Comment (elitucker) [reply src/main/java/views/AcesUp/AcesUp.flt.html]: Fantastic. I think we're good to pull then.

Comment (Lucas-Campos-Davis) [reply src/main/java/views/AcesUp/AcesUp.flt.html]: Good catch. It looks like this has no conflicts with the main branch. So, I say we are good to merge.

**Pull request merged**